packages/frontend/app/routes/peers.$peerId.tsx

```diff
@@ -211,13 +211,7 @@ export default function ViewPeerPage() {
                     error={response?.errors.http.fieldErrors.incomingAuthTokens}
                     description={
                       <>
-                        List of valid tokens to accept when receiving{' '}
-                        <a
-                          className='default-link'
-                          href='https://rafiki.dev/concepts/interledger-protocol/connector/#incoming-http'
-                        >
-                          incoming ILP packets from the peer.
-                        </a>
+                        List of valid tokens to accept when receiving incoming ILP packets from the peer.
```

Comment:
```suggestion
                        List of valid tokens to accept when receiving{' '}
                        <a
                          className='default-link'
                          href='https://rafiki.dev/integration/services/backend-service#interledger-connector'
                        >
                          incoming ILP packets from the peer.
                        </a>
```

Reply:
I had a look at this for the outgoing auth token too and we don't have a great spot other than in the admin docs. There is a table that gives definitions for these things. So either we could provide a sensible header above that table so we can link straight there. Or, I'm left asking myself why we have no other mention of the outgoing auth token? Isn't that something work talking about? Even if it's back in the Interledger connector notes again?

Reply:
Let's leave this without a link for now. We can always go in and add a link later, when there's a better spot for it. I'll make myself a note to follow up on this.